README.md

```diff
@@ -0,0 +1,280 @@
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details> -->
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Add Book]**
+- **[Remove Book]**
+- **[Track progress on your reading]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://tsohle.me/readers-shine)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> How to clone and use this project project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-A working browser application (Google chrome, Mozilla Fire fox, Microsoft edge ...)
+
+-VSCode or any other equivalent code editor
+
+-Node Package Manager (For running, debuging, or installing dependancies to be used)
+
+node and npm:
+
+-Linux
+```sh
+  sudo apt install nodejs npm
+```
+-windows 
+
+[download and install](https://nodejs.org/en/)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Open the command prompt in the created directory.
+
+- On the terminal run this command 
+```sh
+git clone https://github.com/tsohleDev/readers-shine.git
+```
+
+- Go to the repository folder using command prompt
+```sh
+ cd readers-shine
+```
+
+### Install
+
+Install this project with:
+
+```sh
+ run npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+  npm run watch
+```
+-
+
+### Deployment
+
+You can deploy this project using:
+
+- To start run 
+```sh
+  npm run publish
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> All of the collaborators of this project.
+
+👤 **Tsohle Mokhemisi**
+
+- GitHub: [@githubhandle](https://github.com/tsohleDev2)
+- Twitter: [@twitterhandle](https://twitter.com/RealTsohle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tsohle-mokhemisi-3687401b2/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Create the mobile version]**
+<!-- - [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]** -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] Kindly ensure that you fix the the [issues page](../../issues/) broken link as it displays **404** error page when clicking on it. You can replace it with the working link to make your README professional